app/models/roto_scraper.rb

```diff
@@ -46,6 +46,7 @@ def self.go
         previous_date = Date.new(2015, 1, 1)
 
 
+        # SCRAPE QUARTERBACKS
```

Comment:
Comments can be useful for documenting what is happening in code. 

However, when you add a comment like this, it can sometimes be a signal that its time to introduce a method with the same name that describes what it happening:

```diff
- # SCRAPE QUARTERBACKS
+ scrape_quaterback(player, career_stat_rows, game_log_rows)
...
+ def self.scrape_quarterback(player, career_stat_rows, game_log_rows)
+   # Quarterback scraping code goes here
+   ...
+ end
```

Using intention-revealing method names like `scrape_quarterback` instead of comments is one technique for writing self-documenting code. Prefer self-documenting code over commented code.